### PR TITLE
Remove support for old versions of PostgreSQL

### PR DESCRIPTION
**Pull request by jgoizueta**
We aim to support only versions >= 9.5

### CODE REVIEW

**Comment by jgoizueta** [comment]
Please @rafatower @iriberri take a look

File: odbc_fdw.c [hunk 1]
```diff
@@ -527,7 +515,7 @@ getNameQualifierChar(SQLHDBC dbc, StringInfoData *nq_char)
 {
     SQLCHAR name_qualifier_char[2];
 
-#ifdef DUBUG
```
Comment (rafatower) [inline odbc_fdw.c]: "da bug" :D

File: odbc_fdw.c [hunk 2]
```diff
@@ -986,60 +971,6 @@ static ForeignScan* odbcGetForeignPlan(PlannerInfo *root, RelOptInfo *baserel,
...
-    fdwplan = makeNode(FdwPlan);
-    fdwplan->startup_cost = 10;
-    fdwplan->total_cost = 100 + fdwplan->startup_cost;
-    fdwplan->fdw_private = NIL;	/* not used */
-
-#ifdef DEBUG
-    elog(NOTICE, "new total cost: %f", fdwplan->total_cost);
-#endif
-
-    odbcGetTableSize(svr_dsn, svr_driver, svr_host, svr_port, svr_database, svr_schema, svr_table, username, password, sql_count, &table_size);
```
Comment (rafatower) [inline odbc_fdw.c]: not sure whether you want to keep or loose this size/cost estimation logic

File: odbc_fdw.c (hunk 1 above)
Comment (iriberri) [inline odbc_fdw.c]: 🐛

File: odbc_fdw.c (hunk 2 above)
Comment (rafatower) [inline odbc_fdw.c]: it is in the repo history anyway so fine by me

**Comment by rafatower** [comment]
:+1:

File: odbc_fdw.c (hunk 2 above)
Comment (jgoizueta) [inline odbc_fdw.c]: Yes, I'm assuming it's not valid for PG>=9.02 so dismissing it. It'll be nice to rewrite it for PG 9.5

**Pull request merged**